integrationTests/chainSimulator/esdt/issue_test.go

```diff
@@ -36,21 +30,18 @@ func TestChainSimulator_IssueESDTWithPrefix(t *testing.T) {
 		t.Skip("this is not a short test")
 	}
 
-	startTime := time.Now().Unix()
-	roundDurationInMillis := uint64(6000)
 	roundsPerEpoch := core.OptionalUint64{
 		HasValue: true,
 		Value:    20,
 	}
 
-	numOfShards := uint32(1)
 	cs, err := chainSimulator.NewChainSimulator(chainSimulator.ArgsChainSimulator{
 		BypassTxSignatureCheck:      false,
 		TempDir:                     t.TempDir(),
 		PathToInitialConfig:         defaultPathToInitialConfig,
-		NumOfShards:                 numOfShards,
-		GenesisTimestamp:            startTime,
-		RoundDurationInMillis:       roundDurationInMillis,
+		NumOfShards:                 1,
```

Comment:
We should put 3 here. 
With your current implementation, this test might fail. You should revert the common funcs back to their original implementation.